antenna-documentation/src/site/markdown/template-bundle-development.md

```diff
@@ -4,6 +4,24 @@ automatically when no other bundle is configured. This guide will show how to cr
 integrate it in the Antenna workflow run. 
 
 ## Custom Template Bundle
+
+### How to create the template PDF files
+
+First, the template has to be developed with a tool like Microsoft Word / Open Office / Libre Office. The corresponding 
+document will consist of 4 pages which will represent the 4 templates PDF files:
+* cover.pdf
+* copyright.pdf 
+* content.pdf 
+* back.pdf
+
+The page size is A4 at 72 PPI with 595x842 pixels. The cover.pdf file will be the cover page of the attribution document 
+which will contain the project name and version starting at 80x547. The copyright.pdf file will be the second page of 
+the attribution document which will contain the project copyright notes starting at 70x659. The content.pdf file will be
+used several times which will be list the used dependencies with PURL, file name, license and copyright notes. The 
+back.pdf file is the last page of the attribution document which is appended without any changes.  
+
+After the document is finished, each individual page is exported as the respective template PDF file. 
```

Comment:
I believe it's a bit confusing to speak about PDF files before this sentence. The workflow as I understand it is that first do the full design in Word, not knowing / caring that you would export to PDF later. You basically only care about the paper size and the number of pages at this point. And only at the very end I'd explain that you need to export the first page to a PDF file named `cover.pdf`, the second page to a PDF called `copyright.pdf` and so on.

Reply:
I rewrite the section before this sentence without mentioning PDF files.